packages/optimizer/README.md

```diff
@@ -111,6 +112,20 @@ Automatically import any missing AMP Extensions (e.g. amp-carousel).
 - default: `true`
 - used by: [AutoExtensionImport](lib/transformers/AutoExtensionImporter.js)
 
+#### `esmModulesEnabled`
+
+Enables the smaller ESM module version of AMP runtime and components:
```

Comment:
This is a bit ambiguous - is there a way to link to some kind of docs to explain this more?

Reply:
Not yet - we should add one...once there's one.